bee-ledger/src/workers/snapshot/download.rs

```diff
@@ -66,10 +67,16 @@ impl<'a> SourceInformation<'a> {
         Ok(())
     }
 
+    fn index(&self) -> MilestoneIndex {
+        self.delta_header
+            .as_ref()
+            .map_or(self.full_header.sep_index(), SnapshotHeader::sep_index)
```

Comment:
I'm a bit confused, I thought you said in a previous comment that we should sort by ledger_index of full headers ?

Reply:
Sorry, I should have mentioned that: I checked with the code in Hornet again and they sort on the return of this method: https://github.com/gohornet/hornet/blob/ecdb18b0f8ea6a775807e1159e908d28a1ae0363/pkg/snapshot/snapshot_file.go#L270. Which should be the same as the method above (sans the check for `fullHeader`).